sync.py

```diff
@@ -154,19 +155,20 @@ def repos_for_topics(repos_by_topic, topics):
 
 
 class MetaRepoSyncer:
-    def __init__(self, clone_dir, name, submodules, author):
+    def __init__(self, clone_dir, name, submodules, author, sync_depth):
         self.clone_dir = clone_dir
         self.name = name
         self.submodules = submodules
         self.author = author
+        self.sync_depth = sync_depth
 
         self.dir = os.path.join(clone_dir, name)
         self.check_call = functools.partial(subprocess.check_call, cwd=self.dir)
 
     def clone(self, init_submodules=True):
         if not os.path.isdir(os.path.join(self.clone_dir, self.name)):
             logging.info('Cloning meta repository %s', self.name)
-            subprocess.check_call(shlex.split('git clone --depth 1 git@github.com:{}/{}.git'.format(ORGANIZATION, self.name)), cwd=self.clone_dir)
+            subprocess.check_call(shlex.split('git clone --depth {} git@github.com:{}/{}.git'.format(self.sync_depth, ORGANIZATION, self.name)), cwd=self.clone_dir)
             if init_submodules and self._has_submodules():
                 self.check_call(shlex.split('git submodule update --depth 1 --init --jobs 8'))
```

Comment:
Should the same depth apply to this? Should we have a different parameter?

Reply:
Same depth. There shouldn't be any need for a separate one.